src/pages/blog.js

```diff
@@ -1,34 +1,69 @@
 import Head from 'next/head';
 // import { Blogs } from '../components/Global'
 
-export default function Blog({ articles }) {
+export const getStaticProps = async () => {
+  const res = await fetch('https://dev.to/api/articles?username=webxdao');
+  const data = await res.json();
+
+  return {
+    props: {
+      data,
+    },
+  };
+};
+
+const blogs = ({ data }) => {
   return (
     <>
       <Head>
         <title>Blog | WebXDAO</title>
       </Head>
-      {/* <section className='text-white text-center bg-[#00007F]'>
-        <div className='px-20 py-20'>
-          <h1 className='font-bold text-5xl antialiased'>Check out our blog posts</h1>
-          <div className='mt-6 text-xl font-light text-true-gray-500 antialiased'>
-            Read our articles from <span className='font-mono'>Dev.to</span>
+      <section className='py-8'>
+
+        <div className='container max-w-5xl mx-auto'>
+          <h1 className='w-full my-2 text-4xl font-bold leading-tight text-center text-white'>
+            Blogs
+          </h1>
+          <div className='w-full mb-4'>
```

Comment:
```suggestion
          <div className="w-full mb-4">
```

Hi please, kindly suggest you to update the className with double quote: ` " `